src/drive/web/modules/search/AppBarSearch.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+
+import { BarSearch } from 'components/Bar'
+import BarSearchAutosuggest from './components/BarSearchAutosuggest'
```

Comment:
nit: Without the `./`

Reply:
`components` without `./` import global component VS with `./` import local component from search module

Reply:
for clarity, I changed the imports to alias paths

Reply:
Oh my bad!

Reply:
No worries, the comment allowed me to make my imports clearer ;)